repair/row_level.cc

```diff
@@ -2195,6 +2195,15 @@ static future<> repair_put_row_diff_with_rpc_stream_handler(
                             current_rows,
                             std::move(row_opt)).handle_exception([from, repair_meta_id, sink, &error] (std::exception_ptr ep) mutable {
                         rlogger.warn("repair_put_row_diff_with_rpc_stream_handler: from={} repair_meta_id={} error={}", from, repair_meta_id, ep);
+                        if (try_catch<replica::no_such_column_family>(ep)) {
+                            return sink(repair_stream_cmd::table_dropped).then([sink] () mutable {
+                                return sink.flush();
+                            }).then([sink] () mutable {
+                                return sink.close();
+                            }).then([sink, ep = std::move(ep)] () {
+                                return make_exception_future<stop_iteration>(ep);
```

Comment:
Do we want to return this exception? Didn't we already handle this by sending `table_dropped` above?

Reply:
Right, sorry, I messed up the sides on which the exception is handled